src/browser/browser.js

```diff
@@ -255,8 +258,22 @@ define((require, exports, module) => {
         onClose: item => webViewersCursor.update(closeTab(item)),
         onOpen: item => webViewersCursor.update(addTab(item))
       })
-    ]);
-  });
+    ]),
+    DOM.div({
```

Comment:
Is there any reason why we would not add these nodes only when update is available, instead of hiding it when it's not ?